packages/website/generate-pages.js

```diff
@@ -15,45 +11,56 @@ const commandLineArgs = require('command-line-args');
 const processConfig = require('./process-config');
 const generatePages = require('./page-generator');
 
-const cwd = process.cwd();
-
-const cliOptions = [{ name: 'config', type: String }];
-const args = commandLineArgs(cliOptions, { camelCase: true });
-
-const loadConfig = () => {
-  if (args.config) {
-    const configPath = path.resolve(cwd, args.config);
-    return JSON.parse(fse.readFileSync(configPath, 'utf-8'));
-  }
-
-  return {};
-};
-
-const { packagesPaths, docsPath, useManifests } = processConfig(
-  cwd,
-  loadConfig(),
-);
-
-const pagesPath = path.resolve(__dirname, './pages');
-const componentsPath = path.resolve(__dirname, './components/page-templates');
-
-const pagesList = generatePages(
-  packagesPaths,
-  docsPath,
-  pagesPath,
-  componentsPath,
-  {
-    useManifests,
-  },
-);
-const { packages, docs, metaData } = pagesList;
-
-fse.writeFileSync(
-  path.resolve(__dirname, 'data/pages-list.json'),
-  JSON.stringify({ packages, docs }, undefined, 2),
-);
-
-fse.writeFileSync(
-  path.resolve(__dirname, 'data/packages-data.json'),
-  JSON.stringify({ metaData }, undefined, 2),
-);
+module.exports = () => {
+  const cwd = process.cwd();
+
+  const cliOptions = [{ name: 'config', type: String }];
+  const args = commandLineArgs(cliOptions, { camelCase: true });
+
+  const loadConfig = () => {
+    const defaultConfigPath = path.join(cwd, 'docs.config.js');
+
+    let configPath;
+    if (args.config) {
+      configPath = path.resolve(cwd, args.config);
+    } else if (fse.existsSync(defaultConfigPath)) {
+      configPath = defaultConfigPath;
+    }
+
+    if (configPath) {
+      // eslint-disable-next-line
```

Comment:
Please specify a rule you are disabling rather than just disabling the next line

Also, a description of why above the disable is nice.